classes/controller/AdminController.php

```diff
@@ -3146,7 +3146,7 @@ public function getList(
             $select_shop = ', shop.name as shop_name ';
         }
 
-        if ($this->multishop_context && Shop::isTableAssociated($this->table) && !empty($this->className)) {
+        if ($this->multishop_context && Shop::isTableAssociated($this->table) && !empty($this->className) && null !== $this->_join) {
```

Comment:
Why do you change this behavior ?
Did you find a case that induce some problems ?
Is it linked with 
```php
$test_join = !preg_match('#`?' . preg_quote(_DB_PREFIX_ . $this->table . '_shop') . '`? *sa#', $this->_join);
```
?